tests/test-sentry.php

```diff
@@ -1,22 +1,68 @@
 <?php
 
+use Pressbooks\Sentry;
+
+/**
+ * @group sentry
+ */
+
 class SentryTest extends \WP_UnitTestCase {
-	use utilsTrait;
 
 	/**
-	 * Test sentry init function
-	 * It should be false since we are using mock credentials for Sentry
-	 *
-	 * @return \PHPUnit\Framework\Assert
-	 *
+	 * @var Sentry
+	 */
+	protected $sentry;
+
+	/**
+	 * @var Pressbooks\DataCollector\User
+	 */
+	protected $user;
+
+	/**
+	 * Test setup
+	 */
+	public function setUp() {
+		parent::setUp();
+		$this->user = $this->factory()->user->create_and_get( [ 'role' => 'contributor' ] );
+		$this->sentry = new Sentry();
+	}
+
+	/**
+	 * Test init Sentry function to test instance returned.
+	 */
+	public function test_getInstance() {
```

Comment:
Additional negative test cases would increase the robustness of the feature.
Example:
1. Missing environment variable -> application still functioning as expected
2. No user signed in -> application still logs with user as null

Test to ensure the errors are actually tracked by Sentry would be a great validation of the feature working with simulated errors.

Reply:
Makes sense! I will add those.